app/javascript/components/track/Trophies.tsx

```diff
@@ -181,8 +181,8 @@ const UnrevealedTrophy = ({
   updateTrophy: (trophy: Trophy) => void
 }): JSX.Element => {
   const [showError, setShowError] = useState(false)
-  const [mutation] = useMutation(
-    () => {
+  const { mutate: mutation } = useMutation(
+    async () => {
```

Comment:
No `await` was used, so why use `async`?

Reply:
Yes, these changes were made using jscodeshift. I used a script provided by TanStack Query for a simpler migration